operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerEventSource.java

```diff
@@ -122,13 +143,16 @@ private synchronized void onAddOrUpdate(String operation, R newObject, Runnable
     }
   }
 
-  @Override
-  public void onDelete(R resource, boolean b) {
-    if (log.isDebugEnabled()) {
-      log.debug("On delete event received for resource id: {}", ResourceID.fromResource(resource));
-    }
-    super.onDelete(resource, b);
-    propagateEvent(resource);
+  private boolean temporalCacheHasResourceWithVersionAs(R resource) {
```

Comment:
```suggestion
  private boolean temporaryCacheHasResourceWithSameVersionAs(R resource) {
```

Reply:
changed this separatelly in a commit, because here might break the build.